skopt/tests/test_searchcv.py

```diff
@@ -78,6 +83,22 @@ def test_searchcv_runs(surrogate, n_jobs, n_points):
     assert_greater(opt.score(X_test, y_test), 0.9)
 
 
+def test_parallel_cv():
+    """
+    Test whether two parallel jobs acutally work faster than
+    one job.
+    """
+
+    start_time = time.time()
+    test_searchcv_runs(surrogate='gp', n_jobs=1, n_points=1, cv=10)
+    time_one_job = start_time - time.time()
+    start_time = time.time()
+    test_searchcv_runs(surrogate='gp', n_jobs=2, n_points=1, cv=10)
```

Comment:
Does anyone have any experience with calling a `test_` function from another one? I've not seen it done before and wondering if it will confuse pytest/coverage reporting. If no one has any experience then let's leave it like this and find out :)

Reply:
I do this in one of my other projects over and and over - works like a charm!